### DISCUSSION TITLE
Factura global duda de Ñ

### DISCUSSION THREAD

**Discussion by Sandragvb** [bambucode/tfacturaelectronica#176] [General]
Uso el Pac de Comercio digital, pero no logro hacer que se timbre la factura global por la Ñ
<cfdi:InformacionGlobal Año="2022" Meses="05" Periodicidad="01"/>
El XML si esta bien formado y si lo paso mediante la aplicación POSTMAN pasa correcto.
Pero al ejecutar el HTTP.HTTPMethod obtengo este error 
errmsg: no pudo cargar XML '+' is an unexpected token. The expected token is '='. Line 2, position 58.

 llamadoExitoso:=HTTP.HTTPMethod('POST', fWsTimbradoComercio+'/timbre4/timbrarV5');
    if (llamadoExitoso) and (ProcesaErrorPac(HTTP.Headers)) then

¿Alguien que me pueda ayudar?

**Comment by lcarrasco** [comment]
Hola @Sandragvb verifica que en la petición POST estés mandando un encoding correcto de UTF8. En el caso de la aplicación de ejemplo que timbra con Ecodex funciona correctamente.
